typescript/utils/src/arrays.ts

```diff
@@ -0,0 +1,16 @@
+interface Sliceable {
```

Comment:
The rest of the files here in `utils/src/*` is code that was previously in the `utils/src/utils.ts` file or the SDK, just reorganized into specific files.